airbyte-integrations/connectors/source-tvmaze-schedule/source_tvmaze_schedule/tvmaze_schedule.yaml

```diff
@@ -0,0 +1,72 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.tvmaze.com"
+    http_method: "GET"
+    request_options_provider:
+      request_parameters:
+        country: |
+          {{
+            config['domestic_schedule_country_code']
+            if options['name'] == 'domestic' 
+            else config['web_schedule_country_code'].replace('global', '')
+            if options['name'] == 'web' 
+            else ''
+          }}
```

Comment:
not sure if this the best frmat, why not overwrite the value directly in the stream to remove all this `if` clauses?

Reply:
I tried overriding this in the stream's `$options`, but it seems that it doesn't accept the `InterpolatedString` type, as it just used the string `{{ config['domestic_schedule_country_code'] }}` as the request parameter